### PR TITLE
Install Ruby

### PR DESCRIPTION
**Pull request by Kyle1668**
Make sure the follow tasks are done.

 - [ ] Change README.md to include changes
 - [ ] Test it with `make docker`
 - [ ] Wait for review
 - [ ] [Create Release Note](https://opszero.typeform.com/to/JMnLYH) once task is reviewed.

### CODE REVIEW

File: ami/base.sh [hunk 1]
```diff
@@ -27,6 +27,17 @@ sudo apt-get install -y apt-transport-https \
                         wget \
                         zsh
 
+# Ruby
+echo 'ruby begin_________________________________________________'
```
Comment (abhiyerra) [inline ami/base.sh]: Remove the echos

File: ami/docker [hunk 2]
```diff
@@ -0,0 +1,16 @@
+sudo apt-get install -y \
```
Comment (abhiyerra) [inline ami/docker:1]: Why is this file checked in?

Comment (Kyle1668) [reply ami/docker:1]: Error. Will fix that.

File: ami/base.sh [hunk 3]
```diff
@@ -28,6 +28,17 @@ sudo apt-get install -y apt-transport-https \
                         zsh
 
 
+# Ruby
+cd
```
Comment (abhiyerra) [inline ami/base.sh:32]: `pushd /tmp`

File: ami/base.sh [hunk 4]
```diff
@@ -28,6 +28,17 @@ sudo apt-get install -y apt-transport-https \
...
+wget http://ftp.ruby-lang.org/pub/ruby/2.5/ruby-2.5.0.tar.gz
+tar -xzvf ruby-2.5.0.tar.gz
+cd ruby-2.5.0/
```
Comment (abhiyerra) [inline ami/base.sh:35]: `pushd ruby-2.5.0`

File: ami/base.sh [hunk 5]
```diff
@@ -28,6 +28,17 @@ sudo apt-get install -y apt-transport-https \
...
+./configure
+make
+sudo make install
+ruby -v
+cd
```
Comment (abhiyerra) [inline ami/base.sh:40]: `popd; popd`

Comment (Kyle1668) [reply ami/base.sh:40]: Why popd twice?

Comment (abhiyerra) [reply ami/base.sh:40]: Figure out what popd does.

**Comment by abhiyerra** [comment]
@Kyle1668 What is the status of this?

**Pull request merged**